cmake/Modules/FindEigen3.cmake

```diff
@@ -0,0 +1,97 @@
+# - Try to find Eigen3 lib
```

Comment:
Its needed because `Ubuntu 14.04` isnt being able to find `Eigen3` by default although its installed in `/usr/include`. Its not needed for 16.04 or Mac.